javascript/operations.js

```diff
@@ -440,7 +454,7 @@ export default {
     operate(operation, () => {
       const { key, value, type } = operation
       const storage = type === 'session' ? sessionStorage : localStorage
-      storage.setItem(key, value || '')
+      storage.setItem(key, value != null ? value : '')
```

Comment:
I'm pretty sure that storage assignment implicitly calls `toString` but forcing a string before invocation is 👍🏻

Reply:
It does, but `null` get set to `"null"` (string) rather than blank. Which we may or may not want.

Reply:
I don't see value in the *string* `'null'`. If someone wants it, they should pass `"null"` as a string.